lambdas/data-migration2/tests/test-granulesAndFiles.js

```diff
@@ -833,6 +1065,108 @@ test.serial('migrateGranulesAndFiles processes multiple granules and files', asy
   });
 });
 
+test.serial('migrateGranulesAndFiles updates multiple granules and files when migrateAndOverwrite is set', async (t) => {
```

Comment:
Is there any value in a test that tries to run `migrateGranulesAndFiles` against a set of granules where some have already been migrated and some have not? It's maybe very redundant, just trying to think of scenarios that haven't been explicitly tested.

Reply:
That seems entirely valid, and not all that complicated to add to this test.

Reply:
How about 6628dcfbb6

Reply:
Nice, thanks! Not strictly necessary but it feels better to have I suppose.